reactComponents/src/searchbar/redux/tokenSearchSlice.ts

```diff
@@ -8,16 +8,14 @@ import {TokenSearchState} from "./types";
 
 export const setPair = createAsyncThunk(
   'token/setPair',
-  async ({ selectedPair }:any) => {
-    console.log("setPair")
+  async ({ selectedPair }:any) => {    
     return selectedPair;
   }
 );
 
 export const resetSearchOnNewExchange = createAsyncThunk(
   'token/searchReset',
-  async (searchString:any, thunkAPI:any) => {
-    console.log("resetSearchOnNewExchange")
+  async (searchString:any, thunkAPI:any) => {    
```

Comment:
take out params if params if not needed

Reply:
Regarding the backend logic, we still need to improve.

Reply:
Removed.